sql-plugin/src/main/scala/com/nvidia/spark/rapids/AbstractGpuJoinIterator.scala

```diff
@@ -36,7 +36,7 @@ import org.apache.spark.sql.vectorized.ColumnarBatch
 abstract class AbstractGpuJoinIterator(
     gatherNvtxName: String,
     targetSize: Long,
-    opTime: GpuMetric,
+    val opTime: GpuMetric,
```

Comment:
I made it a val so that anything that inherits from this can get access to it.  I thought it was a little cleaner when I did it, but I am not sold on it and I am happy to change it back.

Reply:
I'm fine with it as-is, mostly was curious on the reasoning since it didn't seem necessary as of now.